### ISSUE TITLE
rich text editor shorten URLs to own instance in a wrong way

### ISSUE THREAD

**Issue by gannebamm** [GeoNode/geonode#8787]
## Expected Behavior

If I use links for other resources on my instance inside the metadata abstracts rich text editor the links do follow my given URL.

## Actual Behavior
The links will get shortened in the wrong way. See this example: https://development.demo.geonode.org/documents/214

## Steps to Reproduce the Problem

  1. create a resource
  2. use the rich text metadata abstract editor and put in links to other resources (eg documents)
  3. see how they will get shortened in a wrong way (`documents/` is missing)
 
## Specifications

  - GeoNode version: 3.3.1
  - Installation method (manual, GeoNode Docker, SPCGeoNode Docker): https://development.demo.geonode.org
  - Platform: https://development.demo.geonode.org
  - Additional details:

**Comment by giohappy** [comment]
@gannebamm it looks like a bug in Tinymce. Look at the video below. Even if I change the links in source code mode and the source code is reopened they appear truncated.

![tinymce_bug_links](https://user-images.githubusercontent.com/571129/153404071-ea80d2fc-118c-4f68-b702-8c4737a820c7.gif)

**Comment by gannebamm** [comment]
You are right. The only way I was able to fix it was to fiddle around in the PostgreSQL but than the other fields will not get regenerated like CSW abstract and whatever else there is.

**Comment by giohappy** [comment]
@gannebamm I suspect we will have to investigate if it's a gun in the specific version of TinyMCE.

**Comment by giohappy** [comment]
Closing as part of a repository cleanup: this issue was last updated before 2025-01-01. Not a judgment on the report itself — feel free to reopen or file a fresh issue against `main` if it's still relevant. Thanks for the report.
